src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+    private double area;
```

Comment:
Why do you need this field if anyway you each time calculate a new value? Let's remove it